apps/avatax/bruno/2_Update_delivery_method.bru

```diff
@@ -0,0 +1,81 @@
+meta {
+  name: 2. Update delivery method
+  type: graphql
+  seq: 3
+}
+
+post {
+  url: {{API_URL}}
+  body: graphql
+  auth: bearer
+}
+
+auth:bearer {
+  token: {{storefrontUserToken}}
+}
+
+body:graphql {
+  mutation checkoutDeliveryMethodUpdate($checkoutId: ID!, $deliveryMethodId: ID!) {
+    checkoutDeliveryMethodUpdate(
+      id: $checkoutId
+      deliveryMethodId: $deliveryMethodId
+    ) {
+      errors {
+        ...CheckoutErrorFragment
+      }
+      checkout {
+        id
+        lines {
+          totalPrice {
+            gross {
+              ...Money
+            }
+            tax {
+              ...Money
+            }
+          }
+        }
+        shippingPrice {
+          gross {
+            ...Money
+          }
+          tax {
+            ...Money
+          }
+        }
+        totalPrice {
+            gross {
+                ...Money
+            }
+          	tax {
+            	...Money
+          	}
+        }
+      }
+    }
+  }
+  
+  fragment CheckoutErrorFragment on CheckoutError {
+    message
+    field
+    code
+  }
+  
+  fragment Money on Money {
+    currency
+    amount
+  }
+  
+}
+
+body:graphql:vars {
+  {
+      "checkoutId": "{{checkoutId}}",
+      "deliveryMethodId": "{{deliveryMethodId}}"
+  }
+}
+
+vars:post-response {
+  ~checkoutAmount: res.body.data.checkoutDeliveryMethodUpdate.checkout.totalPrice.gross.amount
```

Comment:
**Question:** do we need it?

If this won't be extracted (checkbox in unchecked under vars tab):
![CleanShot 2024-02-19 at 08 22 25@2x](https://github.com/saleor/apps/assets/9116238/8b1a3bba-0abc-405a-85d0-dbffc5426742)